website/docs/stitch-combining-schemas.md

```diff
@@ -97,48 +94,112 @@ export interface SubschemaConfig {
 
 ```js
 import { buildSchema } from 'graphql';
 import { linkToExecutor } from '@graphql-tools/links';
 
-export const chirpSubschema = {
-  schema: buildSchema(chirpTypeDefs),
-  executor: linkToExecutor(chirpServiceLink),
+export const postsSubschema = {
+  schema: buildSchema(postsServiceTypeDefs),
+  executor: linkToExecutor(postsServiceLink),
 };
 ```
 
-The remote schema's type definitions string may be obtained via introspection (see `introspectSchema`) or through your own internal protocol.
+* `schema`: this is a non-executable schema representing the remote API. The remote schema's SDL (schema definition language) may be obtained through a dedicated service (similar to the [federation service spec](https://www.apollographql.com/docs/federation/federation-spec/#query_service)), or using [introspection](/docs/remote-schemas/#introspectschemaexecutor-context). Note that not all GraphQL servers enable introspection, and those that do will not include custom directives.
+* `executor`: is a generic method that performs requests to a remote schema. You may [write your own](/docs/remote-schemas#creating-an-executor), or use the `linkToExecutor` helper to wrap a [link package](https://www.npmjs.com/package/apollo-link-http). Subschema config uses the `executor` for query and mutation operations, and accepts a `subscriber` function for subscription operations.
+
+See [remote schemas](/docs/remote-schemas/) documentation for more related tools and information.
+
+## Duplicate types
+
+Stitching has two strategies for handling types duplicated across subschemas: a merge strategy (default), and an older binary strategy. You may select between these strategies using the `mergeTypes` option.
 
-An executor is a generic method that performs requests to a remote schema. You may write your own, or use the `linkToExecutor` helper to wrap a link package such as [apollo-link-http](https://www.apollographql.com/docs/link/links/http/). Subschema config accepts an `executor` option for query and mutation operations, and a `subscriber` function for subscription operations. See the [remote schema](/docs/remote-schemas/) docs for more information.
+### Merged types
 
-_**For pre-version 5:** the old method of using [makeRemoteExecutableSchema](/docs/remote-schemas/) to create a local proxy of a remote schema still works. However, it adds an additional layer of delegation that can be avoided by sending settings directly to `stitchSchemas` via SubschemaConfig._
+Duplicate type names are merged by default in GraphQL Tools v7. That means objects, interfaces, and input objects with the same name will have their fields consolidated from across subschemas, and unions will consolidate all member types. The combined gateway schema will then smartly delegate portions of a request to the proper origin schema(s). See [type merging guide](/docs/stitch-type-merging/) for a comprehensive overview.
```

Comment:
Are enum sets merged as well? I've never tried to have an enum deviate across schemas...

Reply:
I think they are!

Reply:
https://github.com/ardatan/graphql-tools/blob/master/packages/stitch/src/mergeCandidates.ts#L57

Reply:
Can't say that I wrote any tests for that bit of code. ;)